modules/bigquery-dataset/README.md

```diff
@@ -54,6 +54,70 @@ module "bigquery-dataset" {
 # tftest modules=1 resources=2 inventory=iam.yaml
 ```
 
+### Authorized Views, Datasets, and Routines
```

Comment:
What about having a complete example with the raw resources and the authorized resources? Similar to the one in the resource [doc](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_dataset#example-usage---bigquery-dataset-authorized-dataset)?

Reply:
Probably a diagram may also help to understand.

Reply:
I've added a more complete example in the latest PR